orbital-mechanics/TheRealLecture1/SpacecraftDynamicsWeek1r;$.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Not sure why this file is in the lecture 10 branch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hello, would it be possible to close this branch? Ive updated lecture 10 with a new branch off of main rather than lecture 1. Thanks